src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -15,6 +15,10 @@
 import seedu.address.model.module.ZoomLink;
 import seedu.address.model.module.grade.Assignment;
 import seedu.address.model.tag.Tag;
+import seedu.address.model.task.Criterion;
+import seedu.address.model.task.Date;
+import seedu.address.model.task.Priority;
+import seedu.address.model.task.TaskName;
```

Comment:
Perhaps we can consider abstracting out all Name related classes, such as ModuleName, ContactName and TaskName into a common Name class. This would prevent repetition of code.

Reply:
Yes, we could use inheritance here to have better abstraction (no repetition code).